app/controllers/suggestion.rb

```diff
@@ -0,0 +1,77 @@
+get '/suggestions' do
+  @comments = Comment.all
+  @suggestions = Suggestion.all #define instance variable for view
+  erb :'suggestions/index' #show all suggestions view (index)
+end
+
+get '/suggestions/new' do
+  @suggestion = Suggestion.new
+  erb :'suggestions/new' #show new suggestions view
+end
+
+post '/suggestions' do
+  # user = User.find_by(id: @user)
+  # @suggestion = user.suggestions.new(params[:suggestion])
+   # binding.pry
+   @suggestion = Suggestion.new(params[:suggestion])
```

Comment:
This line should work 👍  Why? Because the `name` attributes on the inputs are _nested_ inside of `suggestion`. Take a look at the `input` elements and the `textarea` element [here](https://github.com/nyc-red-pandas-2017/suggestion-box-crud/blob/1c4d8c6ba2f1c4989d9cd1c180371592be7078d9/app/views/suggestions/new.erb#L52-L62), specifically at the `name` attribute for each of those elements. Note how their names are "nested" inside of `suggestion`.